### PR TITLE
docs/convert cloud solution design to markdown

### PR DESCRIPTION
**Pull request by el-veee**
Upload the Food Remedy Cloud solution design as a markdown file to GitHub

### CODE REVIEW

**Comment by el-veee** [comment]
The tables in Markdown are barely readable, but they look correct when rendered.

**Review by johnlct — APPROVED** [review]
The document looks good. Great work

File: handover/food-remedy-api/foodremedy-cloud-solution-design.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+| Google Compute Engine          	| Lower-level compute platform that supports container-optimised virtual machines 	| Most control over hardware. Can use container-optimzied VM images.                                                                                                                                         	| Must manage VMs which host containers. Must manage server security, networking, hardening, patching etc.                                                                       	|
+|                                	|                                                                                 	|                                                                                                                                                                                                            	|                                                                                                                                                                                	|
```
Comment (deasta) [inline handover/food-remedy-api/foodremedy-cloud-solution-design.md:27]: Extra unused table row here.

**Review by deasta — COMMENTED** [review]
Done very quickly. Thanks for this and your other documentation contributions.

**Pull request merged**